artifacts/definitions/Windows/Detection/Mutants.yaml

```diff
@@ -8,10 +8,11 @@ parameters:
   - name: processRegex
     description: A regex applied to process names.
     default: .
-    type: regex
```

Comment:
can we keep the type here?

Reply:
I didnt notice that - I edited a version < 0.6.2. Just updated